sdk/authorization/arm-authorization/test/authorization_operations_test.spec.ts

```diff
@@ -63,7 +61,7 @@ describe("Authorization test", () => {
     await recorder.stop();
   });
 
-  it("roleDefinitions create test", async function () {
+  it.skip("roleDefinitions create test", async function () {
```

Comment:
why skip this?

Reply:
.only need add extra config when running test cases, so I remove .only and add .skip

Reply:
@mpodwysocki do you think we should enable allowOnly by default? I feel like this is a behaviour breaking?

Reply:
Yes, that's fine, we can enable it

Reply:
Do we need to allow `.only` for a few management packages, or all of them? If it is just a few of them, I prefer setting `allowOnly` for individual packages. Because it is easy to commit and merge a `.only` that is meant for local testing if we allow for all.

Reply:
Wait I am confused. If `.only` worked locally, why would it fail in CI?

Reply:
> Do we need to allow `.only` for a few management packages, or all of them? If it is just a few of them, I prefer setting `allowOnly` for individual packages. Because it is easy to commit and merge a `.only` that is meant for local testing if we allow for all.

That's a good point, I think in this case, we do try to skip some cases because they can't pass. I am fine with this change now. Thanks all for the help.